src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -479,12 +485,32 @@ redeemADA :: WalletWebMode ssc m => SendActions m -> CWalletRedeem -> m CTx
 redeemADA sendActions CWalletRedeem {..} = do
     let base64rify = T.replace "-" "+" . T.replace "_" "/"
     seedBs <- either
-        (\e -> throwM $ Internal ("Seed is invalid base64 string: " <> toText e))
+        (\e -> throwM $ Internal ("Seed is invalid base64(url) string: " <> toText e))
```

Comment:
@Martoon-00 we don't - but this is just to be sure we didn't f*** up before. Redeem ada code should be in base64 format. I had conversation with @volhovM and he told me that just to be sure there wasn't any mistake done before - we are converting base64 into non-url format before dumping and creating genesis block. This should meet those condition said and implemented from RSCoin -> Cardano Genesis generator https://github.com/input-output-hk/cardano-sl/pull/163/files#diff-fc6ae95807983ca40cedce171a560ba2R41